wiki/Performance_points/id.md

```diff
@@ -0,0 +1,132 @@
+
+Pada 16 Januari 2021, perubahan dilakukan pada sistem ppv2 yang bertujuan memberikan pp secara lebih akurat pada aspek yang lebih sulit pada suatu map. Perubahan ini sebagian besar dilakukan berkat bantuan berbagai pemain seperti [Xexxar](https://osu.ppy.sh/users/2773526) dan [StanR](https://osu.ppy.sh/users/7217455). Detail perubahan dijelaskan secara lebih detail di [post berita yang berhubungan](https://osu.ppy.sh/home/news/2021-01-14-performance-points-updates). Secara singkat, poin-poin perhatian utama pada pembaruan tersebut adalah sebagai berikut.
+
+- Mengenalkan faktor pengali untuk perolehan pp tambahan dari map dengan approach rate 11 berdasarkan durasi beatmap tersebut
+- Mengenalkan faktor pengali untuk pengurangan pp yang terpengaruhi oleh mod [No Fail](/wiki/Game_modifier/No_Fail) berdasarkan jumlah miss pada sebuah permainan
+- Mengenalkan faktor pengali untuk pengurangan pp yang terpengaruhi oleh mod [Spun Out](/wiki/Game_modifier/Spun_Out) berdasarkan jumlah spinner pada beatmap tersebut
+- Menyesuaikan nilai dari kerugian pp oleh miss untuk berdampak lebih kecil pada map lebih lama dengan kombo yang lebih tinggi
+- Mengurangi perolehan pp untuk [akurasi](/wiki/Accuracy) yang lebih rendah
+
+ppv2 pada saat ini bersifat aktif dengan pembaruan yang dirilis di [changelognya](https://osu.ppy.sh/p/changelog?category=pp).
+
+## Penghitungan
+
+Performance point itu dihitung secara berat berdasarkan pada tingkat kesusahan beatmap yang dihitung, yang mana ditentukan oleh algoritma yang unik yang dibangun untuk setiap [game mode](/wiki/Game_mode) masing-masing.
+
+Tingkat kesulitan dari beatmap yang dimainkan menentukan jumlah akhir pp dari skor mereka. Secara matematis, rumus penghitungan pp memiliki empat nilai inti: **[aim](#aim)**, **[speed](#speed)**, **[akurasi](#accuracy)**, dan **[strain](#strain)**. Semuanya kemudian digabungkan dalam ukuran yang berbeda-beda untuk menghasilkan skor keseluruhan yang terkait dengan tingkat [kesulitan](/wiki/Beatmap/Difficulty) dan performa pemain dalam beatmap tersebut.
+
+Setelah itu, skor "dibobotkan" satu sama lain untuk memastikan bahwa hanya skor-skor terbaik yang didapatkan pemain yang paling berpengaruh terhadap  ranking performance point mereka secara keseluruhan. Diketahui sebagai [*sistem pembobotan*](#weightage-system), sistem ini bertujuan untuk mencegah perolehan skor dengan pp kecil secara cepat dan berulang pada beatmap yang mudah dengan mengurangi jumlah pp yang sebenarnya diperoleh berdasarkan skor terbaik lainnya yang dimiliki pemain tersebut.
+
+*Catatan: Sejumlah kecil pp bonus diberikan berdasarkan pada jumlah map Ranked yang sudah kamu dapatkan skor.*
+
+### Sistem pembobotan
+
+Sistem pembobotan adalah rumus sederhana yang digunakan setelah penghitungan performance point penuh sebuah permainan. Rumus ini digunakan untuk mengurangi jumlah pp yang diperoleh berdasarkan penempatan permainan tersebut dalam skor-skor terbaik pemain.
```

Comment:
missing an empty line on the line after this

Reply:
my bad